capstone-rs/src/capstone.rs

```diff
@@ -165,6 +165,18 @@ impl Capstone {
         }
     }
 
+    /// Creates an instance of DisasmIter structure
+    ///
+    pub fn get_disasm_iter<'a>(&'a self) -> DisasmIter<'a> {
+        DisasmIter {
+            insn: unsafe { cs_malloc(self.csh()) },
```

Comment:
we should panic on allocation failures, per usual Rust norms (currently this will fail silently)

Reply:
I couldn't find an example I could follow in this repo for the handling a malloc issue. Can you please point me to an example if what I pushed now is not the right approach? Thnx!

Reply:
Nothing too crazy--if the return for `cs_malloc()` is NULL, then just call `panic!("cs_malloc() failed")`. :-)